packages/cycle-scripts-one-fits-all/configs/babelrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "presets": [
+        [ "env", {
+            "targets": {
+                "browsers": ["last 2 versions"],
+                "uglify": true
```

Comment:
if you've pulled this out of package.json then I think the user will need to be able to override as with webpack.conf. 
I'm not familiar with the syntax but that would appear to read that uglify is applied to dev builds too which will not be wanted.